tests/components/laundrify/test_config_flow.py

```diff
@@ -0,0 +1,193 @@
+"""Test the laundrify config flow."""
+from unittest.mock import patch
+
+from laundrify_aio import exceptions
+
+from homeassistant.components.laundrify.const import DOMAIN
+from homeassistant.config_entries import SOURCE_REAUTH, SOURCE_USER, ConfigEntryState
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_CODE, CONF_SOURCE
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from . import (
+    _patch_laundrify_exchange_code,
+    _patch_laundrify_get_account_id,
+    _patch_laundrify_get_machines,
+    _patch_laundrify_validate_token,
+    create_entry,
+)
+from .const import VALID_ACCESS_TOKEN, VALID_AUTH_CODE, VALID_USER_INPUT
+
+
+def _patch_setup_entry():
```

Comment:
same thing here

Reply:
Fixed in e54dbf5bf2c4d9def50dfb20a89c9f35a78e9622